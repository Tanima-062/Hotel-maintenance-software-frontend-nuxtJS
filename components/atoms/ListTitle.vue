<template>
  <div class="flex items-center justify-between bg-menuColor w-full p-3">
    <slot />
  </div>
</template>
<script lang="ts">
import { Vue, Component } from 'nuxt-property-decorator'
@Component({})
export default class extends Vue {}
</script>
