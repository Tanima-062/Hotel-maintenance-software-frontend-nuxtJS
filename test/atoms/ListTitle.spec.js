import { shallowMount } from '@vue/test-utils'
import ListTitle from '@/components/atoms/ListTitle.vue'

describe('Atoms ListTitle Check', () => {
  test('ListTitle Test', () => {
    const wrapper = shallowMount(ListTitle)

    // check if it is a vue element
    expect(wrapper.vm).toBeTruthy()

    // check Snapshot
    expect(wrapper.element).toMatchSnapshot()
  })
})
