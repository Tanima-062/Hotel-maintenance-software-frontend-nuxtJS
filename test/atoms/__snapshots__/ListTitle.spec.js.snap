// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atoms ListTitle Check ListTitle Test 1`] = `
<div
  class="flex items-center justify-between bg-menuColor w-full p-3"
/>
`;
